scripts/installer.sh

```diff
@@ -7,14 +7,6 @@
 
 set -eu
 
-# Ensure that this script runs with the default umask for Linux. In practice,
-# this means that files created by this script (such as keyring files) will be
-# created with 644 permissions. This ensures that keyrings and other files
-# created by this script are readable by installers on systems where the
-# umask is set to a more restrictive value.
-# See https://github.com/tailscale/tailscale/issues/15133
-umask 022
```

Comment:
Do we know whether there are other chmod invocations here that we should also check?

Reply:
This covers the two places that I found where we create a file that needs to be later read by installer tool like apt, but is unreadable by it on systems with restrictive umask.